erpnext/projects/doctype/task/task.json

```diff
@@ -138,6 +140,7 @@
    "fieldname": "parent_task",
    "fieldtype": "Link",
    "ignore_user_permissions": 1,
+   "in_list_view": 1,
```

Comment:
This kinda caused an error on Task list view. Ref: https://discuss.frappe.io/t/databases-error-in-tasks-after-update/110119

@s-aga-r